ViewModels/TagsViewModel.cs

```diff
@@ -0,0 +1,58 @@
+﻿using DolphinDynamicInputTexture.Data;
+using DolphinDynamicInputTextureCreator.Models.Suggestions;
+using DolphinDynamicInputTextureCreator.Other;
+using DolphinDynamicInputTextureCreator.ViewModels.Commands;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows.Input;
+
+namespace DolphinDynamicInputTextureCreator.ViewModels
+{
+    class TagsViewModel : Other.PropertyChangedBase
+    {
+        /// <summary>
+        /// The Tags mapped in this pack
+        /// </summary>
+        public UICollection<Tag> Tags
+        {
+            get => _tags;
+            set
+            {
+                _tags = value;
+                TagsSuggestions.SetTargetList(Tags);
+                OnPropertyChanged(nameof(Tags));
+            }
+        }
+        private UICollection<Tag> _tags;
+
+        /// <summary>
+        /// possible key Suggestions for the Selected emulated device.
```

Comment:
Summary probably needs to be updated here :)

Reply:
have changed everything as suggested. :)